import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATIProcessor.java

```diff
@@ -293,27 +528,29 @@ protected Document getDocument(String fileName) throws ParserConfigurationExcept
         }
     }
 
-    public NodeList getNodeListFromXpath(Document xDoc, String xPathExpresion)  {
+    public NodeList getNodeListFromXpath(Document xDoc, String xPathExpresion) {
 
         XPath xPath = XPathFactory.newInstance().newXPath();
         try {
-            Node node= (Node)xPath.compile(xPathExpresion).evaluate(xDoc,XPathConstants.NODE);
+            Node node = (Node) xPath.compile(xPathExpresion).evaluate(xDoc, XPathConstants.NODE);
```

Comment:
The variable node is not used. Please remove

Reply:
done